ps_checkout.php

```diff
@@ -153,7 +153,7 @@ public function __construct()
 
         // We cannot use the const VERSION because the const is not computed by addons marketplace
         // when the zip is uploaded
-        $this->version = '2.15.0';
+        $this->version = self::VERSION;
```

Comment:
```suggestion
        $this->version = '2.16.0';
```
We cannot use the const VERSION because the const is not computed by addons marketplace when the zip is uploaded

Reply:
Sniff ! :(